lzy-api/src/main/proto/ai/lzy/v1/workflow/workflow-service.proto

```diff
@@ -96,6 +96,7 @@ message GraphStatusResponse {
   message Completed {}
   message Failed {
     string description = 1;
+    string failedTask = 2;
```

Comment:
let's rename it to the `failedTaskName` and add one more field -- `failedTaskId`

Reply:
task id - is uniq id, task name - xz :)